tutorial/chapter1.md

```diff
@@ -98,10 +102,10 @@ In this example there are some features of Klepsydra that will be introduced and
 
 * The ```kpsr::Service``` was introduced in the [kpsr-tutorial - chapter 1](https://github.com/klepsydra-technologies/kpsr-tutorial/blob/master/tutorials/chapter1.md) and it is used in this example (by means of inheritance) to create two custom services:
 
-   * ```SimpleWriteService```, which is a service that resizes and publishes the resized images
-   * ```kpsr_tutorial::ImageProcessingService```, which is a service with the next features:
-      * it has a consumer of these images published
-      * it has a publisher of these images (after that ```simpleWriteService``` publishes them)
+   * ```SimpleWriteService```, which is a service that reads images from a folder
```

Comment:
Change to FileImageStreamingService. Remove the SimpleWriteService and delete other unused classes/files.

Reply:
I removed SimpleWriteService from the chapter1. I don't know what delete more (about classes/files).

Reply:
Done